app/forms/candidate_interface/becoming_a_teacher_form.rb

```diff
@@ -2,18 +2,28 @@ module CandidateInterface
   class BecomingATeacherForm
     include ActiveModel::Model
 
-    attr_accessor :becoming_a_teacher
+    attr_accessor :becoming_a_teacher, :single_personal_statement
 
-    validates :becoming_a_teacher,
-              word_count: { maximum: 600 },
-              presence: true
+    alias single_personal_statement? single_personal_statement
+
+    validates :becoming_a_teacher, presence: true
```

Comment:
We’re going to have to customise this error message between the old and new personal statements somehow... 😩 

The old one says this when nothing is entered, which doesn’t make sense for the new merged question:
<img width="833" alt="Screenshot 2023-03-13 at 14 24 10" src="https://user-images.githubusercontent.com/30665/224730159-3ee3903f-cd6d-4528-9e9e-c3d4a22de26a.png">

Alternatively, could follow the approach from this experimental PR and return the user to the application page (and mark section as incomplete) if submitted when blank? #8019

Reply:
I've added custom validations (for now!)